### DISCUSSION TITLE
Disable streak option

### DISCUSSION THREAD

**Discussion by Doranov** [dailydotdev/daily#1143] [Feature Requests]
I know why you're using the "streak" functionality, and I'm sure it helps people to meet their goals. For me it has the opposite effect as I prefer intrinsic over extrinsic motivation. And on principle I don't like addictive "features". Another user filed a request to be able to set the active days, and the word "frustrating" was used. I already quit Brilliant.org and it was in part because they had a streak feature. I don't want to quit daily.dev because so far I've had a lot of fun reading the articles. It would be really nice if I had the option to turn the streak feature off.

**Comment by nimrodkra** [comment]
Hey thanks for the feedback! This version is still an experimental MVP. We
will definitely add more configurations if it ends up being helpful to
people (including the ability to opt out of it). We don’t have a specific
timeline for it, but we will definitely take this feedback into account

**Comment by sywesk** [comment]
Bouncing back on this, now that streaks are quite well set-up: We need to have a way to ignore everything about streaks. It's extremely frustrating to be guilt-tripped because I didn't read an article during a day. Like, imagine being in full focus mode on a hard task, and loosing a "streak" because of that, how frustrating. Plus your system isn't tolerant at all which makes it even more frustrating to loose a streak, like when i didn't open my laptop during a bank holiday.

I know startups are addicted to gamification to increase retention & time spent, but we don't need points to stay on the app, we just need quality tech news. If you don't do it yourself, i'll mod daily.dev to remove it myself and make the modification available

**Reply by Doranov** [reply]
I already quit daily.dev because of the streak feature. It seemed all the focus was on the streak feature instead of the articles. If ever that mod becomes available, I'll no doubt return to daily.dev. The content was good, but the streak feature killed it for me.

**Reply by idoshamun** [reply]
We already have this capability to disable streak, it's in the customize menu after clicking your profile image

**Comment by idoshamun** [comment]
This feature already exists, please see my comment: https://github.com/dailydotdev/daily/discussions/1143#discussioncomment-11130826

**Reply by FDelporte** [reply]
Thanks but you need to add the option to disable this from the streak warning popup so people can delete this the first time it pops up. I didn't find how to disable it and was going to delete the plugin from my browser for this reason until I found this question...
